### PR TITLE
Support bootstrapping and configuration as non-root user (refs #62)

### PR DESCRIPTION
**Pull request by tomster**

### CODE REVIEW

File: bsdploy/roles/jails_host/defaults/main.yml [hunk 1]
```diff
@@ -23,3 +23,5 @@ ploy_jail_host_pkg_repository: |
         mirror_type: "srv",
         enabled: yes
     }
+ploy_root_user_name: "{{ploy_user | default('root')}}"
+ploy_root_home_path: "{{ '/' if ploy_root_user_name == 'root' else '/usr/home/' }}{{ploy_root_user_name}}"
```
Comment (fschulze) [inline bsdploy/roles/jails_host/defaults/main.yml]: the root home is '/root' not '/'

Comment (fschulze) [inline bsdploy/roles/jails_host/defaults/main.yml]: ah, duh, forget it

Comment (tomster) [inline bsdploy/roles/jails_host/defaults/main.yml]: :-)

Comment (tomster) [inline bsdploy/roles/jails_host/defaults/main.yml]: it's the root-home-root!!

File: bsdploy/roles/jails_host/tasks/main.yml [hunk 2]
```diff
@@ -121,7 +121,13 @@
   command: "tar -x -C /usr/jails/flavours/bsdploy_base --chroot --exclude '+*' -f /var/cache/pkg/All/pkg.txz creates=/usr/jails/flavours/bsdploy_base/usr/local/sbin/pkg"
 
 - name: The .ssh directory for root in bsdploy_base flavour
-  file: dest=/usr/jails/flavours/bsdploy_base/root/.ssh state=directory mode=0600 owner=root group=wheel
+  file:
+    dest: "/usr/jails/flavours/bsdploy_base/{{ploy_root_home_path}}/.ssh"
+    state: directory
+    mode: 0600
```
Comment (fschulze) [inline bsdploy/roles/jails_host/tasks/main.yml]: This might not work, yaml converts unquoted numbers. You should test and possibly use quotes.

Comment (fschulze) [inline bsdploy/roles/jails_host/tasks/main.yml]: Yaml is especially weird with numbers starting with 0. Search for yaml warts or gotchas.

Comment (fschulze) [inline bsdploy/roles/jails_host/tasks/main.yml]: It worked before, because it was one long string, now it's separate.

Comment (fschulze) [inline bsdploy/roles/jails_host/tasks/main.yml]: I just checked my playbooks and I always have it in quotes, pretty sure it's required to work.

**Comment by tomster** [comment]
i was now able to configure a virtualbox instance as ``tomster`` using the fix and docs from above.

unless @fschulze finds a reason not to, i'm for merging this onto master

**Comment by fschulze** [comment]
Merged manually after squasing the commits.

**Pull request closed without merging**